core/txpool/subpool.go

```diff
@@ -86,6 +86,11 @@ type PendingFilter struct {
 	OnlyBlobTxs  bool // Return only blob transactions (block blob-space filling)
 }
 
+type TxMeta struct {
```

Comment:
I would rename this to `TxMetadata` and the method in the interface to `GetMetadata`

Reply:
fixed